data/public-datasets-repository.md

```diff
@@ -106,5 +106,17 @@ Jobs and notebooks have access to a read-only directory that is mounted at `/dat
       style="text-align:left"><a href="https://github.com/NVlabs/stylegan">https://github.com/NVlabs/stylegan</a> 
         </td>
     </tr>
+    <tr>
+      <td style="text-align:left">
+        <p><em><b>Tiny ImageNet</b></em>
+        </p>
+        <p><code>/datasets/tiny-imagenet-200</code>
+        </p>
+      </td>
+      <td style="text-align:left">The Tiny ImageNet is a subset of the ImageNet dataset and contains 200 classes with each class having         500 training images, 50 validation images and 50 test images. It is the default course project of the Stanford CS231n Class.            </td>
```

Comment:
hey @digantamisra98, i just now found that i was tagged in this! thank you for submitting this. could you just remove the extra spaces before `500` and after `Class.`? looks great to me otherwise. thanks again!

Reply:
@jaredscheib Hi, I had unfortunately deleted the fork. Should I submit a fresh PR for this again ?
Thanks!

Reply:
if you're willing to, i'd love to merge it -- thank you!

Reply:
Sure, I'll do that. Thanks!

Reply:
@jaredscheib Just checked that the content of my PR has already been added on the file in the master branch,. Hence I'm closing this PR.

Reply:
interesting -- good to know! thank you again~